app/src/androidTest/java/org/simple/clinic/medicalhistory/MedicalHistorySyncAndroidTest.kt

```diff
@@ -44,11 +45,14 @@ class MedicalHistorySyncAndroidTest : BaseSyncCoordinatorAndroidTest<MedicalHist
 
   val authenticationRule = AuthenticationRule()
 
+  val rxErrorsRule = RxErrorsRule()
+
   val registerPatientRule = RegisterPatientRule(patientUuid = UUID.randomUUID())
 
   @get:Rule
   val ruleChain = RuleChain
       .outerRule(authenticationRule)
+      .around(rxErrorsRule)
```

Comment:
Same as earlier comment about `rxErrorsRule` being the innermost rule.

Reply:
Same answer as earlier comment about `rxErrorsRule` being the innermost rule.